requirements.txt

```diff
@@ -1,6 +1,6 @@
 fastapi==0.63.0
 httpx==0.16.1
-reasoner-pydantic@git+git://github.com/ranking-agent/reasoner-pydantic@trapi-v1.1
+reasoner-pydantic@git+git://github.com/ranking-agent/reasoner-pydantic@v1.1.1
```

Comment:
Let's reference this on PyPI whenever possible. There is a `reasoner-pydantic==1.1.1`.